app/services/groups/add_users_service.rb

```diff
@@ -52,7 +52,11 @@ def persist(call)
     def after_perform(call)
       Groups::CreateInheritedRolesService
         .new(model, current_user: user, contract_class:)
-        .call(user_ids: params[:ids], message: params[:message])
+        .call(
+          user_ids: params[:ids],
+          message: params[:message],
+          send_notifications: params.fetch(:send_notifications, true)
```

Comment:
Not necessary as `after_perform` is called from `BaseServices::BaseContracted.perform`, and the call is surrounded by `service_context(send_notifications: (params || {}).fetch(:send_notifications, true))`: the `:send_notification` param has already been taken into account. (see another comment below).

I tried removing it and the tests still pass.

Reply:
Well, thinking about it, it may be helpful in one case: it will prevent displaying a warning if `send_notifications: false` is given to `Groups::AddUsersService.call` because `Groups::CreateInheritedRolesService.call` would try to use `send_notifications: true` and it will display a warning if already set.

Reply:
maybe the default value for `send_notifications` in all the calls should be `nil` instead of `true`, so that the `Journal::NotificationConfiguration` can ignore the value if it's nil, and try to change it only if it's explicitly `false` or `true`. 

This way the warning message would not be displayed in the case I described above (where value is set to `false` and then is not forwarded when calling subservices).

code could look like this:
```ruby
    def with(send_notifications, &)
      if send_notifications.nil?
        yield
      elsif already_set?
        log_warning(send_notifications)
        yield
      else
        with_first(send_notifications, &)
      end
    end
```